d2/src/main/java/com/linkedin/d2/balancer/servers/ZooKeeperServer.java

```diff
@@ -60,6 +60,11 @@ public String getConnectString() {
     return _store.getConnectString();
   }
 
+  @Override
+  public AnnounceMode getAnnounceMode() {
+    return AnnounceMode.STATIC_OLD_SR_ONLY;
```

Comment:
According to `AnnounceMode`, this could be 
`AnnounceMode.STATIC_OLD_SR_ONLY` or `AnnounceMode.DYNAMIC_OLD_SR_ONLY` ?

Reply:
DYNAMIC is for IndisZooKeeperServer, as only that class can listen lix and switch dynamically.

Reply:
I see. So `DYNAMIC` is only for `IndisZooKeeperServer`